src/chart/sparkline.js

```diff
@@ -0,0 +1,117 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.charts.sparkline = function() {
+
+        // creates an array with four elements, representing the high, low, open and close
+        // values of the given array
+        function highLowOpenClose(data) {
+            var xValueAccessor = sparkline.xValue(),
+                yValueAccessor = sparkline.yValue();
+
+            var high = d3.max(data, yValueAccessor);
+            var low = d3.min(data, yValueAccessor);
+
+            function elementWithYValue(value) {
+                return data.filter(function(d) {
+                    return yValueAccessor(d) === value;
+                })[0];
+            }
+
+            return [{
+                    x: xValueAccessor(data[0]),
+                    y: yValueAccessor(data[0])
+                }, {
+                    x: xValueAccessor(elementWithYValue(high)),
+                    y: high
+                }, {
+                    x: xValueAccessor(elementWithYValue(low)),
+                    y: low
+                }, {
+                    x: xValueAccessor(data[data.length - 1]),
+                    y: yValueAccessor(data[data.length - 1])
+                }];
+        }
+
+        var xScale = fc.scale.dateTime();
+        var yScale = d3.scale.linear();
+        var line = fc.series.line();
+
+        // configure the point series to render the data from the
+        // highLowOpenClose function
+        var point = fc.series.point()
+            .xValue(function(d) { return d.x; })
+            .yValue(function(d) { return d.y; })
+            .decorate(function(sel) {
+                sel.attr('class', function(d, i) {
```

Comment:
Yes, but I think it would be more messy ...

```
sel.classed({
  open: function(d, i) { return i === 0; },
  high: function(d, i) { return i === 1; },
  low: function(d, i) { return i === 2; },
  close: function(d, i) { return i === 3; },
})
```